docs/whats-new.mdx

```diff
@@ -14,6 +14,8 @@ You can expect this page to update regularly (at least monthly). We'll include t
 
 ## December 2024
 
+- 2024-12-20 - Released [Traffic Policy macros](/traffic-policy/macros), which enable you to encode or decode query strings, Base64 data, and JSON. This simplifies handling [action result variables](/traffic-policy/variables/action/) for OAuth and [custom response](/traffic-policy/actions/custom-response/#action-result-variables) actions.
```

Comment:
```suggestion
- 2024-12-20 - Released [Traffic Policy macros](/traffic-policy/macros) that enable you to encode or decode query strings, Base64 data, and JSON. This simplifies handling [action result variables](/traffic-policy/variables/action/) for OAuth and [custom response](/traffic-policy/actions/custom-response/#action-result-variables) actions.
```

Reply:
One small tweak to suggest! We've already had macros for a while, and as a whole, they do more that de/encodin, so just want to clarify that these new ones are part of a larger set.

Reply:
Thank you!